change/@microsoft-teams-js-db4c1ae0-ce9d-45a5-a3f1-2f9c8d825f1d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "add code comment for dialog.submit",
+  "packageName": "@microsoft/teams-js",
+  "email": "email not defined",
```

Comment:
Your email address associated with your github account should probably be here? Not sure why it isn't